scripts/run_tracker.sh

```diff
@@ -98,7 +98,7 @@ while [ $EXPECTED_SUBSCRIBERS -gt 0 ] || [ $NR_TRACKERS -lt 4 ]; do
     mkdir -p $STATEDIR
     ln -s $OUTPUT_DIR/bootstraptribler.txt $STATEDIR/bootstraptribler.txt
     # Do not daemonize the process as we want to wait for all of them to die at the end of this script
-    twistd --pidfile tracker_$TRACKER_PORT.pid -n $EXTRA_ARGS --logfile="$OUTPUT_DIR/tracker_out_$TRACKER_PORT.log" tracker --port $TRACKER_PORT --crypto $TRACKER_CRYPTO $EXTRA_TRACKER_ARGS --statedir=$STATEDIR &
+    twistd --pidfile $STATEDIR/tracker_$TRACKER_PORT.pid -n $EXTRA_ARGS --logfile="$OUTPUT_DIR/tracker_out_$TRACKER_PORT.log" tracker --port $TRACKER_PORT --crypto $TRACKER_CRYPTO $EXTRA_TRACKER_ARGS --statedir=$STATEDIR &
```

Comment:
This is to make sure that the gumby experiment doesn't litter my tribler directory with a bunch of random pid files.